Games/types/Mafia/items/Whiskey.js

```diff
@@ -26,16 +26,19 @@ module.exports = class Sedative extends Item {
             }
 
             if (reveal && cursed)
-              this.game.queueAlert(
-                `${this.actor.name} accidentally spiked their own drink with a sedative!`
+              this.actor.queueAlert(
+                `You couldn't resist drinking all that whiskey yourself...`
               );
             else if (reveal && !cursed)
-              this.game.queueAlert(
-                `${this.actor.name} spikes ${this.target.name}'s drink with a sedative!`
+              this.actor.queueAlert(
```

Comment:
why does actor need alert for who they roleblock?

Reply:
This is like snowball, previously everyone got an alert but now only Actor and target gets alerts